hedera-node/hedera-evm-api/build.gradle.kts

```diff
@@ -33,4 +33,8 @@ dependencies {
         exclude("com.google.code.findbugs", "jsr305")
     }
     implementation(libs.javax.inject)
+    implementation(libs.tuweni.units) {
```

Comment:
The dependency on units only comes from test code.  Do we really have to encode our test dependencies into the production module definitions?

Reply:
It's needed for `HederaBlockValues` which is exposed in `src/main/java`

Reply:
We should change the return value for HederaBlockValues then to remove this dependency.  It's not an interface requirement and we can return `Bytes.EMPTY` instead of `UInt256.ZERO` - they both mean the same and the former brings along less dependencies.

Reply:
DONE